src/components/application_manager/rpc_plugins/vehicle_info_plugin/include/vehicle_info_plugin/commands/mobile/subscribe_vehicle_data_request.h

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
why there is a vi_subscribe_vehicle_data_request.h in sdl rpc plugin?
what's the purpose/difference between subscribe_vehicle_data_request and vi_subscribe_vehicle_data_request?

similarly, there is  vi_unsubscribe_vehicle_data_request

Reply:
@yang1070  vi_subscribe_vehicle_data_request.h should be in vehicle_info_plugin 

subscribe_vehicle_data_request(response) - mobile commands
vi_subscribe_vehicle_data_request(response) - hmi commands 

all hmi commands start with the name of the interface (in this case it is - VI (Vehicle Info))